app/templates/admin/createSingleEvent.html

```diff
@@ -65,9 +65,9 @@ <h1 id="title">{{page_title}}</h1>
     <div class="row p-2">
       <div class="col-md-6 col-sm-12 ">
         {% if eventData.id %}
-          <input type="hidden" name="id" value="{{eventData.id}}" >
+          <input type="hidden" name="eventId" value="{{eventData.id}}" >
```

Comment:
You can't make this change without changing everywhere that it is used. In this case, creating new events and editing events won't work, because it expects `id` in the event data. If you want to avoid fixing that functionality you should use `id` everywhere instead of `eventId`. If you really want `eventId`, you need to find all of the event-related logic that uses `id` and convert those as well.